server/lib/orcasite/radio/detection.ex

```diff
@@ -84,24 +84,13 @@ defmodule Orcasite.Radio.Detection do
   end
 
   policies do
-    bypass actor_attribute_equals(:admin, true) do
-      authorize_if always()
-    end
-
-    bypass action_type(:read) do
-      authorize_if always()
-    end
-
-    bypass action(:seed) do
-      authorize_if always()
-    end
-
-    bypass action(:submit_detection) do
-      authorize_if always()
-    end
-
-    bypass action(:submit_machine_detection) do
-      authorize_if actor_attribute_equals(:detection_bot, true)
+    bypass always() do
+      authorize_if AshOban.Checks.AshObanInteraction
+      authorize_if actor_attribute_equals(:admin, true)
+      authorize_if action_type(:read)
+      authorize_if action(:seed)
+      authorize_if action(:submit_detection)
+      authorize_if action(:submit_machine_detection)
```

Comment:
Question: Do we really still need separate actions for submit_detection vs submit_machine_detection?

Reply:
I think so, as they're slightly different. The machine one calculates the different fields based on the timestamp, sets the listener count to 0, and the non-machine one accepts the player_offset and playlist_timestamp directly. The intent is also clearer about when one is used over the other (the machine one requires an API key to associate which user/org is submitting it).

Reply:
Makes sense, so maybe rename `submit_detection` to `submit_human_detection` then?